bin/printAlertStream.py

```diff
@@ -9,9 +9,34 @@
 from __future__ import print_function
 import argparse
 import sys
+import os
 from lsst.alert.stream import alertConsumer
 
 
+def msg_text(message):
+    """Remove postage stamp cutouts from an alert message.
+    """
+    message_text = {k: message[k] for k in message if k not in ['cutoutDifference', 'cutoutTemplate']}
```

Comment:
In several places there are hard coded keys like this.  It makes me a little concerned that if the schema changes, it will break.  Is there a way to either figure out these keys from the schema or to have it configurable in a parent class so that we only have to change things in one place?